domains/transfer-agreements/TransferAgreements.API/API/Data/ApplicationDbContext.cs

```diff
@@ -1,17 +1,19 @@
+using Audit.EntityFramework;
 using Microsoft.EntityFrameworkCore;
 
 namespace API.Data;
 
-public class ApplicationDbContext : DbContext
+public class ApplicationDbContext : AuditDbContext
 {
-    public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
-        : base(options)
+    public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
     {
     }
 
     public DbSet<TransferAgreement> TransferAgreements { get; set; }
+    public DbSet<TransferAgreementAudit> TransferAgreementAudits { get; set; }
 
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
+
     }
```

Comment:
Why override this method, when it does nothing?

Reply:
It was there from the beginning. But it is removed now. Thanks for the heads up.